challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+    input.map {|x| x.capitalize}
```

Comment:
There's a shorthand syntax when we just call one method on the block variable:

```ruby
input.map(&:capitalize)
```
This means: call `capitalize` on each item in the array, without needing to name the element as we iterate through them ("x"). It's more "declarative" and generally preferred.